Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -186,13 +186,10 @@ open class MessageSizeCalculator: CellSizeCalculator {
     }
 
     internal func labelSize(for attributedText: NSAttributedString, considering maxWidth: CGFloat) -> CGSize {
-        let estimatedHeight = attributedText.height(considering: maxWidth)
-        let estimatedWidth = attributedText.width(considering: estimatedHeight)
+        let constraintBox = CGSize(width: maxWidth, height: .greatestFiniteMagnitude)
+        let rect = attributedText.boundingRect(with: constraintBox, options: [.usesLineFragmentOrigin, .usesFontLeading], context: nil).integral
```

Comment:
@zhongwuzw Why does this work? 🤔 I think this breaks for small messages like `"lol"`?